### ISSUE TITLE
exec_poll redeclare  error due to lib/installer.php 'include lib/poller.php' 

### ISSUE THREAD

**Issue by ddb4github** [Cacti/cacti#2140]
> PHP  Fatal error:  Cannot redeclare exec_poll() (previously declared in /opt/IBM/cacti/lib/poller.php:28) in /opt/IBM/cacti/lib/poller.php on line 52

Shold use "include_once", like 
`include_once(dirname(__FILE__) . '/../lib/poller.php');
`

**Comment by netniV** [comment]
Can you give more context to this.  Do you have the full stack trace that should also be reported with it?

**Comment by netniV** [comment]
So I'm going to guess that you got this error whilst installing.

```
$ grep "lib/installer.php" * -R --color=always | grep -v locales/
cli/install_cacti.php:include_once($config['base_path'] . '/lib/installer.php');
install/install.php:    include_once('../lib/installer.php');
install/background.php:include_once($config['base_path'] . '/lib/installer.php');
install/step_json.php:include('lib/installer.php');
install/install.js: * lib/installer.php                                       *
lib/installer.php:       * lib/installer.php                                       *
```
This shows that the installer library is only included by a select few scripts.  However, one of the dependencies, may be including it so it's used more than once.  That's possible due to a recent change to api_device.php to fix #2126

**Comment by netniV** [comment]
This has now been committed to the development code.
